systers_portal/templates/account/email.html

```diff
@@ -0,0 +1,63 @@
+
+{% extends "base.html" %}
+
+{% load account %}
+
+{% block title %} - Email{% endblock %}
+
+{% load crispy_forms_tags %}
+
+{% block content %}
+
+<h1>E-mail Addresses</h1>
+{% if user.emailaddress_set.all %}
+  <p>The following e-mail addresses are associated with your account:</p>
+  <form  action="{% url 'account_email' %}" method="post">
+    {% csrf_token %}
+    <fieldset class="blockLabels">
+      {% for emailaddress in user.emailaddress_set.all %}
+      <div class="ctrlHolder">
+        <label for="email_radio_{{forloop.counter}}" class="{% if emailaddress.primary %}primary_email{%endif%}">
```

Comment:
I see in some places you have extra space inside `{{ }}` and `{% %}`. Please follow the same pattern for the rest of Django template blocks.